laravel-app/app/Http/Controllers/FollowController.php

```diff
@@ -20,27 +20,33 @@ public function show(User $user): View
         ]);
     }
 
-    public function update(User $user): RedirectResponse
+    public function update(User $user): JsonResponse
     {
         /** @var User $authUser */
         $authUser = auth()->user();
 
         $authUser->following()->attach($user);
 
-        $success = ['success' => 'Successfully followed ' . $user->username];
+        $message = 'Successfully followed ' . $user->username;
 
-        return redirect()->back()->with($success);
+        return response()->json([
+            'status' => 'success',
+            'message' => $message,
+        ], 200);
     }
 
-    public function destroy(User $user): RedirectResponse
+    public function destroy(User $user): JsonResponse
```

Comment:
Add comment!

Reply:
done sir, added function comment. Thank you